jest.config.js

```diff
@@ -14,6 +15,6 @@ export default {
   ],
   globalSetup: './test/setup-global.js',
   setupFilesAfterEnv: [],
-  collectCoverageFrom: ['**/*.js'],
+  collectCoverageFrom: ['**/*.js', '**/*.cjs', '**/*.mjs'],
```

Comment:
Do you have any relevant `.cjs` files in your repo? Just curious 👍🏼

Reply:
Currently not, but could be forgotten when adding a cjs file in the future. This makes sure that coverage is there